src/pages/app-development/starter-kit/project-setup.md

```diff
@@ -0,0 +1,276 @@
+
+1. Use the **Add service** pop-up menu to select and add the other required services.
+
+1. If you are using JWT authentication, unzip the downloaded `config.zip` file. The extracted `config` directory should contain a `certificate_pub.crt` and a `private.key` file. The `private.key` file is required to configure the Commerce Admin.
+
+## Download the workspace configuration file
+
+The console can generate a JSON file that defines the configuration of your workspace. You will use this file to configure the Commerce Admin and to set up the Starter Kit.
+
+To download a `.json` file containing your workspace configuration:
+
+1. Go to the overview page of your workspace.
+
+1. Click the **Download All** button in the top-right corner.
+
+   The `<Workspace-name>.json` file downloads automatically. In this example, the file is named `977AmethystGazelle-1340225-Stage.json`.
+
+## Create an integration in Adobe Commerce
+
+A Commerce integration generates the consumer key, consumer secret, access token, and access token secret that is required to connect to the Starter Kit. It also identifies the available API resources that are needed for the integration.
+
+Use the following steps to create and activate an integration.
+
+1. From the Admin, go to **System** > Extensions > **Integrations**.
+
+1. Click **Add New Integration**. The **New Integration** page displays.
+
+   ![New integration](../../_images/starterkit/add-integration.png)
+
+1. Enter a name for the integration in the **Name** field. Leave the other fields blank.
+
+1. Click **API** under the Basic Settings menu to display the **Available APIs** screen. Change the value of the **Resource Access** drop-down menu to **All**.
+
+1. Click **Integration Info** and enter your Admin password in the **Your Password** field. Click **Save** to return to the Integration page.
+
+1. Click the **Activate** link in the grid.
+
+  ![Activate link](../../_images/starterkit/activate-integration.png)
+
+1. On the next page, click the **Allow** button to display the **Integration Tokens for Extensions** page.
+
+   ![Integration tokens](../../_images/starterkit/secrets.png)
+
+   You will need the integration details (consumer key, consumer secret, access token, and access token secret) to configure the Starter Kit. Copy these values to a safe place and click **Done**.
+
+## Install Adobe I/O Events for Adobe Commerce (Commerce 2.4.4 and 2.4.5 only)
+
+If you are running Adobe Commerce 2.4.4 or 2.4.5, you must install modules to enable eventing, as described in [Install Adobe I/O Events for Adobe Commerce](../../events/installation.md). These modules are installed automatically in subsequent versions of Commerce.
+
+## Starter Kit set-up and onboarding
+
+You are now ready to download the Adobe Commerce Starter Kit and set up your development.
+
+### Download and configure the project
+
+<!-- Note to reviewer: Will the enterprise-integration/starter-kit repo be mirrored? It looks like a private repo -->
```

Comment:
During this beta phase, we are distributing the starter kit as a zip version of the git repo.
Once the beta phase is completed, the intention is to make the starter kit available in the marketplace.